shared/protocol/src/main/java/io/pravega/shared/protocol/netty/WireCommands.java

```diff
@@ -786,6 +786,60 @@ public long getRequestId() {
         }
     }
 
+    @Data
+    public static final class FlushToStorage implements Request, WireCommand {
+        final WireCommandType type = WireCommandType.FLUSH_TO_STORAGE;
+        final int containerId;
+        @ToString.Exclude
+        final String delegationToken;
+        final long requestId;
+
+        @Override
+        public void process(RequestProcessor cp) {
+            ((AdminRequestProcessor) cp).flushToStorage(this);
+        }
+
+        @Override
+        public void writeFields(DataOutput out) throws IOException {
+            out.writeInt(containerId);
+            out.writeUTF(delegationToken == null ? "" : delegationToken);
+            out.writeLong(requestId);
+        }
+
+        public static WireCommand readFrom(ByteBufInputStream in, int length) throws IOException {
+            int containerId = in.readInt();
+            String delegationToken = in.readUTF();
+            long requestId = in.available() >= Long.BYTES ? in.readLong() : -1L;
+            return new FlushToStorage(containerId, delegationToken, requestId);
+        }
+
+        @Override
+        public long getRequestId() {
```

Comment:
This is not needed. Your class is decorated with `@Data` so it automatically generates getters for all the fields.

Reply:
Removed it.